lib/graphql_rails/model/input.rb

```diff
@@ -20,34 +23,20 @@ def graphql_input_type
         )
       end
 
-      def attribute(attribute_name, type: nil, enum: nil, **attribute_options)
-        input_type = attribute_type(attribute_name, type: type, enum: enum, **attribute_options)
-
-        attributes[attribute_name.to_s] = Attributes::InputAttribute.new(
-          attribute_name, type: input_type, **attribute_options
-        )
-      end
-
       private
 
       attr_reader :input_name_suffix, :model_class
 
+      def build_attribute(attribute_name)
+        Attributes::InputAttribute.new(attribute_name, config: self)
+      end
+
       def default_name
         @default_name ||= begin
           suffix = input_name_suffix ? input_name_suffix.to_s.camelize : ''
           "#{model_class.name.split('::').last}#{suffix}Input"
         end
       end
-
-      def attribute_type(attribute_name, type:, enum:, description: nil, **_other)
-        return type unless enum
-
-        BuildEnumType.call(
```

Comment:
logic moved to https://github.com/samesystem/graphql_rails/pull/211/files#diff-e6a5f9d9da086bd050247e5d70ae2d2554deec89cf57fa322ed0fcdff2152c9fR59